src/main.cpp

```diff
@@ -104,6 +104,7 @@ static void lPrintVersion() {
     PrintWithWordBreaks(cpuHelp, 16, TerminalWidth(), stdout);
     printf("    [-D<foo>]\t\t\t\t#define given value when running preprocessor\n");
     printf("    [--dev-stub <filename>]\t\tEmit device-side offload stub functions to file\n");
+    printf("    [--disable-vectorcall]\t\t\t\tDisable vectorcall calling convention.\n");
```

Comment:
This make sense on Windows only, so this needs to be enabled only for Windows.

Reply:
Done